dot/config_integration_test.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // TestExportConfig tests exporting a toml configuration file
-func TestExportConfig(t *testing.T) {
+func TestExportConfigIntegration(t *testing.T) {
```

Comment:
this function probably doesn't need to be exported as it's in a _test file

Reply:
That's a test, not a helper test function